en/docs/tutorials/configure-automatic-deployment.md

```diff
@@ -0,0 +1,105 @@
+# Configure Automatic Deployment
+
+When you develop a Choreo component, you need to commit its source code into a GitHub repository. You can select a repository that Choreo manages or a repository that you own and manage.
```

Comment:
You can't select a repository that Choreo manages right? Should we rephrase it as "You can choose to commit it to a Choreo managed GitHub repository or a repository that you own and manage"

Reply:
Addressed